config/site.example.yml

```diff
@@ -59,13 +59,38 @@ common: &common
       - Bus
       - Lodging
       - Other
+    assign_states:
+      - incomplete: 'requester'
+      - submitted: 'tsp'
+      - approved: 'requester'
+    notify_states:
+      - incomplete:
+          - to: ['requester', 'tsp', 'assistant']
+          - remind_to: 'requester'
+          - remind_after: 5
+      - submitted:
+          - to: ['requester', 'tsp', 'assistant']
+          - remind_to: ''
+          - remind_after: 0
+      - approved:
+          - to: ['requester', 'tsp', 'assistant']
+          - remind_to: 'requester'
+          - remind_after: 5
+      - accepted:
+          - to: ['requester', 'tsp', 'assistant']
+          - remind_to: ''
+          - remind_after: 0
+      - canceled:
+          - to: ['requester', 'tsp', 'assistant']
+          - remind_to: ''
+          - remind_after: 0
```

Comment:
You can use an [anchor](http://yaml.org/spec/1.0/#id2563853) to reduce duplication. And reference to it down below